CRM/Civicase/Hook/SummaryActions/AlterAddCaseAction.php

```diff
@@ -24,7 +24,7 @@ public function __construct() {
   }
 
   /**
-   * Changes add case link with web form if web form is available.
+   * Changes add case action with web form if web form is available.
```

Comment:
Maybe we should update this to say that it removes the popup for the add case action if a webform link is available?